src/graphql/trusted_user_agent.rs

```diff
@@ -73,22 +73,41 @@ impl UserAgentMutation {
         Ok(true)
     }
 
-    /// Removes a trusted user agents, Returns true if the deletion was successful.
+    /// Removes trusted user agents, returning the list of successfully removed user agents.
     #[graphql(guard = "RoleGuard::new(Role::SystemAdministrator)
         .or(RoleGuard::new(Role::SecurityAdministrator))")]
     async fn remove_trusted_user_agents(
         &self,
         ctx: &Context<'_>,
         user_agents: Vec<String>,
-    ) -> Result<bool> {
+    ) -> Result<Vec<String>> {
```

Comment:
Hi octoaide
The return type of the `removeTrustedUserAgents` GraphQL API has been changed, resulting in a breaking change. Could you please add the relevant entry to the ChangeLog?